ALI/ALI.py

```diff
@@ -158,7 +158,7 @@ def __init__(self, parent):
     ScriptedLoadableModule.__init__(self, parent)
     self.parent.title = "ALI"  # TODO: make this more human readable by adding spaces
     self.parent.categories = ["Automated Dental Tools"]  # set categories (folders where the module shows up in the module selector)
-    self.parent.dependencies = []  # TODO: add here list of module names that this module requires
+    self.parent.dependencies = ["SlicerConda"]  # TODO: add here list of module names that this module requires
```

Comment:
This would be the _module_ name `CondaSetUp`, not the extension name. This determines the module load order at Slicer startup.